src/Salt/UserBundle/Controller/SignupController.php

```diff
@@ -65,6 +65,21 @@ public function signupAction(Request $request)
                 $command = new AddUserCommand($targetUser, $encryptedPassword);
                 $this->sendCommand($command);
 
+                // check to see if the env var is set to "true" mailer
+                if ($this->getParameter('USE_MAIL_FEATURE') == 'always-active') {
+                  // $fromEmail = getenv('MAIL_FEATURE_FROM_EMAIL');
+                  $fromEmail = $this->getParameter('MAIL_FEATURE_FROM_EMAIL');
```

Comment:
The parameter should be the lower-case one (`mail_feature_from_email`) defined in `config.yml`

Reply:
Fixed